File: mybatis-plus-join-test/test-springboot3-jdk17/src/main/java/com/yulichang/test/springboot3jdk17/mapper/UserDTOMapper.java
package com.leheyue.test.springboot3jdk17.mapper;

import com.github.leheyue.base.MPJBaseMapper;
import com.leheyue.test.springboot3jdk17.entity.UserDto;
import org.apache.ibatis.annotations.Mapper;

@Mapper
public interface UserDTOMapper extends MPJBaseMapper<UserDto> {
}

File: mybatis-plus-join-test/test-kotlin/src/main/kotlin/com/github/yulichang/test/kt/mapper/UserDTOMapper.kt
package com.github.leheyue.test.kt.mapper

import com.github.leheyue.base.MPJBaseMapper
import com.github.leheyue.test.kt.entity.UserDto
import org.apache.ibatis.annotations.Mapper

@Mapper
interface UserDTOMapper : MPJBaseMapper<UserDto?>
